public/src/components/amounts/AmountsTestsList.tsx

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+import { List, ListItem, makeStyles, Typography } from '@material-ui/core';
+import { red } from '@material-ui/core/colors';
+import {
+  Territories,
+  Territory,
+  AmountsTests,
+  Regions,
+  Countries,
+  CountryOptions,
+  getTargetName,
+} from '../../utils/models';
+import { CreateTestButton } from './CreateTestButton';
+
+const useStyles = makeStyles(({ palette }) => ({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    paddingLeft: '32px',
+    marginBottom: '24px',
+  },
+  header: {
+    margin: '12px',
+    fontSize: '18px',
+  },
+  list: {
+    marginTop: 0,
+    padding: 0,
+    '& > * + *': {
+      marginTop: '8px',
+    },
+  },
+  testButton: {
+    position: 'relative',
+    height: '50px',
+    width: '290px',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    borderRadius: '4px',
+    padding: '0 12px',
+    textTransform: 'uppercase',
+    fontSize: '12px',
+  },
+  testNotSelected: {
+    border: `1px solid ${palette.grey[700]}`,
+    backgroundColor: 'white',
+    color: palette.grey[700],
+    '&:hover': {
+      backgroundColor: palette.grey[300],
+    },
+  },
+  testIsSelected: {
+    border: `1px solid ${palette.grey[700]}`,
+    color: 'white',
+    backgroundColor: palette.grey[700],
+    '&:hover': {
+      color: palette.grey[700],
+      backgroundColor: palette.grey[300],
+    },
+  },
+  liveTestNotSelected: {
+    border: `1px solid ${red[500]}`,
+    backgroundColor: 'white',
+    color: palette.grey[700],
+    '&:hover': {
+      backgroundColor: palette.grey[300],
+    },
+  },
+  liveTestIsSelected: {
+    border: `1px solid ${red[500]}`,
+    color: 'white',
+    backgroundColor: red[500],
+    '&:hover': {
+      color: palette.grey[700],
+      backgroundColor: palette.grey[300],
+    },
+  },
+}));
+
+interface AmountsTestsListProps {
+  tests: AmountsTests;
+  selectedTest: Territory | undefined;
+  onTargetSelected: (target: Territory) => void;
+  create: (selected: CountryOptions) => void;
+}
+
+interface AmountsTestButtonProps {
+  target: Territory;
+}
+
+const regionLabels = Object.keys(Regions);
+const countryLabels = Object.keys(Countries);
```

Comment:
What does "labels" mean here? The keys of these objects are the code I think (as opposed to something designed for humans to read).